src/components/CResourceList.vue

```diff
@@ -181,6 +182,11 @@ export default {
       type: Function,
       required: true,
     },
+
+    rowClass: {
+      type: Function,
```

Comment:
https://stackoverflow.com/questions/53608244/how-to-add-multiple-data-types-for-vuejs-props